src/modules/awake/Awake/Core/Manager.cs

```diff
@@ -272,12 +273,10 @@ internal static IntPtr GetHiddenWindow()
 
         public static Dictionary<string, int> GetDefaultTrayOptions()
         {
-            Dictionary<string, int> optionsList = new Dictionary<string, int>
-            {
-                { "30 minutes", 1800 },
-                { "1 hour", 3600 },
-                { "2 hours", 7200 },
-            };
+            Dictionary<string, int> optionsList = new Dictionary<string, int>();
+            optionsList.Add("30 " + Resources.AWAKE_MINUTES, 1800);
```

Comment:
Why can’t the strings be fully localized. What if a language has minute number style formatting?

Reply:
The strings could be fully localized. 
The reason why I choose this implementation is that the same menu may contain user defined intervals (like '3 hours 11 minutes'), which are now localized in a way that the words hour(s), minute(s) are replaced by their localized versions. So the idea was to have menu entries build up in the same way.